### ISSUE TITLE
making_classifier script hangs with 2400 images in the raw_faces/group_faces/

### ISSUE THREAD

**Issue by jaiatncson7** [abhiksark/Face-Recognition-ImageNet#1]
Hi abhiksark,

I am testing your code with 2400 images the raw_faces/group_faces/. i could successfully run raw_faces_to_aligned_faces.py but when i run making_classifier.py, it just hang in model.fit(emb_array, labels) function call indefinitely. 

The same code works fine for 1920 images in raw_faces. Any idea how to resolve this issue? Please help.

**Comment by abhiksark** [comment]
I am not sure how to solve this.
What was the ram of the system you were working with? Is this a RAM Issue?

**Comment by jaiatncson7** [comment]
I have fixed it by increasing the batch size on classifier training.
